content/docs/06-integrations/00-ubuntu.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Ubuntu'
-metaTitle: 'Ubuntu Operating System'
+metaTitle: 'Ubuntu'
 metaDescription: 'Choosing Ubuntu as an Operating System within the Palette console'

```

Comment:
```suggestion
metaDescription: 'Choose Ubuntu Operating System pack in Palette.'

```

Reply:
What if we did something this for packs? I'm considering the way I perform a search.

If you like it, I could add it to our Pack Layout page. 

If you don't go for it, I'd just suggest 'Choose' and 'Palette'.